services/jd_sensor.h

```diff
@@ -20,6 +20,7 @@ typedef struct {
     const struct sensor_range *ranges;
     bool (*is_present)(void);
     const char *name;
+    void *instancedata; // Add instance data so target profile can pass in configuration information into driver
```

Comment:
this is typically passed in through #defines like `ACC_I2C_ADDR`

Reply:
maybe in that case add ds1810_pre_init(...) function that app.c has to call before using `&temperature_ds18b20` ? I don't like making it non-const, and the lack of type-safety with the void* ...